charts/miggo/values.yaml

```
@@ -312,6 +312,19 @@ miggoRuntime:
     # setting and set a value for resources.limits.memory.
     useGOMEMLIMIT: true
 
+    # -- Specifies which tracers should be enabled in the profiler.
+    # The following tracers are currently supported:
+    #   * perl
+    #   * php
+    #   * python
+    #   * hotspot
+    #   * ruby
+    #   * v8
+    #   * dotnet
+    #   * go
+    #   * labels
+    enabledTracers: [python, hotspot, dotnet, v8, go]
```

We do support certain versions of PHP as well.
```suggestion
    enabledTracers: [php, python, hotspot, dotnet, v8, go]
```